source/runtime_gui.cpp

```diff
@@ -2114,6 +2118,23 @@ void reshade::runtime::draw_gui_settings()
 			modified = true;
 		}
 
+		// only for d3d11, d3d12, Vulkan API and possible hdr swapchains
+		if ((_renderer_id & 0xB000 || _renderer_id & 0xC000 || _renderer_id & 0x20000)
+		 && (_back_buffer_format == reshade::api::format::r10g10b10a2_unorm || _back_buffer_format == reshade::api::format::b10g10r10a2_unorm ||_back_buffer_format == reshade::api::format::r16g16b16a16_float))
+		{
+			if (float &hdr_overlay_brightness = _hdr_overlay.brightness; ImGui::SliderFloat("HDR overlay brightness", &hdr_overlay_brightness, 20.f, 400.f, "%.0f nits"))
+			{
+				// Prevent user from setting the brightness too low
+				hdr_overlay_brightness = std::max(hdr_overlay_brightness, 20.f);
```

Comment:
ImGui has this nice new flag `ImGuiSliderFlags_AlwaysClamp`, which ensures users can't input values below the specified range. That can probably replace this explicit clamp here.